database/migrations/20170531173912_organizations.js

```diff
@@ -0,0 +1,14 @@
+
+exports.up = knex =>
+  knex.schema.createTable( 'organizations', table => {
+    table.increments( 'id' ).primary()
+    table.integer( 'user_id' ).notNullable()
```

Comment:
The other thing about this that's kinda weird is: what does it mean for an organization to have a `user_id` within it? Is this the user that owns the organization? That created it? Who is this person?

If it's the owner, then this could be made more explicit by calling the column `owner_id`, even though the id it contains is an id for a record in the `users` table.